infra/testApi.mjs

```diff
@@ -236,6 +181,113 @@ checkLocalErrorNode(localError7.tree, [0, 1],
 checkLocalErrorNode(localError7.tree, [2],
   'fma', '0.0', '-inf.0', '-inf.0', '+inf.0', '0.0')
 
+// MARK: Alternatives endpoint
+const alt_pcontext = [[[14.97651307489794], 0.12711304680349078]]
+const alts_sync = callAlternatives(FPCoreFormula, alt_pcontext, false);
+const alts_async = callAlternatives(FPCoreFormula, alt_pcontext, true);
+assert.deepEqual(alts_sync, alts_async);
+
+async function callAlternatives(fpcore, p_context, async_huh) {
+  const alternatives_json = await getJSONFor('alternatives', 'POST', JSON.stringify({
+    formula: fpcore, sample: p_context
+  }), async_huh);
+  assertIdAndPath(alternatives_json);
+  checkFelids(alternatives_json, ['command', 'alternatives', 'histories', 'derivations', 'splitpoints', 'job', 'path']);
+  assert.equal(Array.isArray(alternatives_json.alternatives), true);
+  return alternatives_json;
+}
+
+// MARK: Cost endpoint
+async function callCost(fpcore, p_context, async_huh) {
+  const cost_json = await getJSONFor('cost', 'POST', JSON.stringify({
+    formula: fpcore, sample: p_context
+  }), async_huh);
+  assertIdAndPath(cost_json);
+  checkFelids(cost_json, ['command', 'cost', 'job', 'path']);
+  assert.equal(cost_json.cost > 0, true);
+  return cost_json;
+}
+
+// MARK: Translate
+async function callTranslate(fpcore, language) {
```

Comment:
Why have one function if it's just going to branch on the language? Just make two functions for the two endpoints...